pmd-modelica/src/test/java/net/sourceforge/pmd/lang/modelica/ModelicaParsingHelper.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.modelica;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
that looks strange here.... jetbrains notnull

Reply:
Oh yeah Intellij sometimes inserts this out of nowhere... it's a pain